langchain_benchmarks/extraction/tasks/chat_extraction/__init__.py

```diff
@@ -0,0 +1,35 @@
+from langchain.prompts import ChatPromptTemplate as C
```

Comment:
Imo shouldn't use abbreviations here, ide should auto complete to save typing

Reply:
Works for me!